Data/SkillSys/SkillInfo.c

```diff
@@ -3568,6 +3568,11 @@ const struct SkillInfo gSkillInfos[MAX_SKILL_NUM + 1] = {
     },
 #endif
 
+#if (defined(SID_Swarp) && COMMON_SKILL_VALID(SID_Swarp))
+    [SID_Swarp] = {
+        .desc = MSG_SKILL_Swarp,
```

Comment:
seems encounters some error when merging to master branch

Reply:
Weird, Guthub says it's good to merge. Can you paste them here?

Reply:
I mean it is apparently the skill info is lost here, I'm not sure whther there are more mistakes elsewhere, maybe you'd better re-pull a new PR after testing

Reply:
Ah it must have gotten removed accidently when I was merging the conflicts. I'll add it back in and review with my local

Reply:
Okay all good